cvc4autoconfig.h.in

```diff
@@ -43,8 +43,11 @@
 /* Define if `ffs' is supported by the platform. */
 #cmakedefine HAVE_FFS
 
-/* Define to 1 to use libreadline. */
-#cmakedefine01 HAVE_LIBREADLINE
+/* Define to 1 to use libedit. */
+#cmakedefine01 HAVE_LIBEDITLINE
+
+/* Define to 1 to use line editing (in general). */
+#cmakedefine01 HAVE_LINEEDITING
```

Comment:
Not sure if we need to make this distinction since we only have support for libedit now. I have a slight preference for only `HAVE_LIBEDITLINE` but I can live with two defines as well.

Reply:
Done!